sample_files/main.py

```diff
@@ -1,10 +1,13 @@
-from selenium import webdriver
-from webdriver_manager.chrome import ChromeDriverManager
-import logging
+import os
 import time
+import logging
 import requests
+from dotenv import load_dotenv
+from selenium import webdriver
+from selenium.common.exceptions import WebDriverException, NoSuchElementException
+from selenium.webdriver.common.by import By
+from webdriver_manager.chrome import ChromeDriverManager
 from requests.exceptions import RequestException
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There are some unnecessary spaces before the import statements on lines 2 and 4. This could lead to `IndentationError` in Python.
```diff
-  import time
+ import time

-  import requests
+ import requests
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->